python/ray/experimental/workflow/common.py

```diff
@@ -148,10 +148,11 @@ def slugify(value: str, allow_unicode=False) -> str:
 
 
 class Workflow:
-    def __init__(self, workflow_data: WorkflowData):
+    def __init__(self, workflow_data: WorkflowData, prepare_inputs):
```

Comment:
```suggestion
    def __init__(self, workflow_data: WorkflowData, prepare_inputs: bool):
```

Reply:
It's actually a Callable. let me fix this.